vcbuild.bat

```diff
@@ -79,6 +79,7 @@ if /i "%1"=="without-intl"     set i18n_arg=%1&goto arg-ok
 if /i "%1"=="download-all"  set download_arg="--download=all"&goto arg-ok
 if /i "%1"=="ignore-flaky"  set test_args=%test_args% --flaky-tests=dontcare&goto arg-ok
 if /i "%1"=="enable-vtune"  set enable_vtune_arg=1&goto arg-ok
+if /i "%1"=="dll"           set dll=1&goto arg-ok
 
```

Comment:
Need to unset `dll` at https://github.com/nodejs/node/pull/7487/files#diff-f1d059c92a388e475daa1fbc2a88aa3aR41